### PR TITLE
refactor: update qp2p (quinn)

### PR DESCRIPTION
**Pull request by b-zee**

### CODE REVIEW

**Comment by joshuef** [comment]
bors try

File: sn_node/src/bin/sn_node/main.rs [hunk 1]
```diff
@@ -122,19 +122,6 @@ async fn run_node(config: &Config) -> Result<()> {
     let (_node, mut rejoin_network_rx) = loop {
         match start_node(config, join_timeout).await {
             Ok(result) => break result,
-            Err(NodeError::CannotConnectEndpoint(qp2p::EndpointError::Upnp(error))) => {
```
[the remark this answers is not in the document]
Comment (b-zee) [reply sn_node/src/bin/sn_node/main.rs:125]: I've only taken out IGD (UPnP), mostly for simplicity reasons. When I researched this, many routers do not support it or do not have it enabled by default. It's also something I haven't ever come across so far in my sysadmin experience when configuring server software...

[the remark this answers is not in the document]
Comment (joshuef) [reply sn_node/src/bin/sn_node/main.rs:125]: Okay, good to know. We can always re-add as non-qp2p module or something if we need itdown the line.

**Pull request merged**